nat-lab/tests/utils/router/mac_router.py

```diff
@@ -89,15 +74,14 @@ async def create_meshnet_route(self) -> None:
     async def create_vpn_route(self) -> None:
         if self.ip_stack in [IPStack.IPv4, IPStack.IPv4v6]:
             await self._connection.create_process(
-                ["route", "delete", "-inet", "default"]
-            ).execute()
-
-            await self._connection.create_process(
-                ["route", "add", "-inet", "default", "-interface", self._interface_name]
-            ).execute()
-
-            await self._connection.create_process(
-                ["route", "add", "-inet", VPN_SERVER_SUBNET, LINUX_VM_PRIMARY_GATEWAY]
+                [
```

Comment:
what is the point of this route? it redirects 10.0/16 to tunnel interface, meanwhile you need to simulate all traffic being sent to tunnel interface, except for vpn server ip

Reply:
did you mean to do "default" instead of 10.0.0.0/16 maybe?

Reply:
so maybe 0/1 + 128/1 is what you need? this subnet 10.0/16 is too small to capture all traffic(as the vpn route should do)

Reply:
That seems about right, fixed